.github/workflows/lint.yml

```diff
@@ -86,38 +86,3 @@ jobs:
 
           # Run lintrunner except clang-tidy
           lintrunner --force-color --take FLAKE8,MYPY,CLANGFORMAT,NOQA,TYPEIGNORE,NEWLINE,MYPYSTRICT,TABS,SPACES,EXEC,BLACK --all-files
-
```

Comment:
flake8 and clang-format is already covered by lintrunner, so it makes no sense to run it again. Begging for admin power to lift branch protection requirement.